specifications/transactions/travel-rule/README.md

```diff
@@ -0,0 +1,372 @@
+# Off-CHain Travel Rule Exchange
+
+This protocol is used to exchange KYC information about a potential transaction between to parties that will be resolved on-chain upon its completion. This is a dual attestation, preflight protocol. At the end of the protocol, the sender will receive a signature from the recipient so that the Diem Paymnet Network can verify that the tranasfer has been signed off by both the sender and recipient. Some considerations:
```

Comment:
```suggestion
This protocol is used to exchange KYC information about a potential transaction between to parties that will be resolved on-chain upon its completion. This is a dual attestation, preflight protocol. At the end of the protocol, the sender will receive a signature from the recipient so that the Diem Payment Network can verify that the transfer has been signed off by both the sender and recipient. Some considerations:
```